docker/compile/Dockerfile.debian8

```diff
@@ -0,0 +1,31 @@
+FROM openjdk:8-jdk-slim
+
+# This is passed to the heron build command via the --config flag
+ENV TARGET_PLATFORM debian
+ENV bazelVersion 0.5.4
+
+RUN apt-get update && apt-get -y install \
+      automake \
+      build-essential \
+      cmake \
+      curl \
+      libssl-dev \
+      git \
+      libtool \
+      libtool-bin \
+      python \
+      python2.7-dev \
+      software-properties-common \
+      python-setuptools \
+      zip \
+      unzip \
+      wget
+
+ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
```

Comment:
suggest to move this line to beginning. not a blocker.

Reply:
removed this since the image already sets JAVA_HOME.